.envsample

```diff
@@ -0,0 +1 @@
+DATABASE_URL=rename this file to .env and input your database url here
```

Comment:
Awesome job on setting up the database yusuf, I would like to suggest that instead of asking that the .envsample file be renamed, let the user create their own .env file. Also you can add an example of what the .env file should look like in the .envsample file.